mediacapture-streams/MediaStream-MediaElement-preload-none-src.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!-- Copyright © 2016 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
```

Comment:
For this particular PR though, it's less clear to me that this stamp is needed since these are direct-to-wpt, though since authored by a Googler (and reviewed by w3c members), this stamp is still correct and perhaps best to keep even in this case.